util/build_configs/cray-internal/setenv-hpe-cray-ex-x86_64.bash

```diff
@@ -89,10 +89,17 @@ if [ -z "$BUILD_CONFIGS_CALLBACK" ]; then
     export CHPL_COMM_SUBSTRATE=none
     export CHPL_TASKS=qthreads
     export CHPL_LAUNCHER=none
-    export CHPL_LIBFABRIC=libfabric
+    export CHPL_LIBFABRIC=system
     export CHPL_LLVM=llvm       # llvm requires py27 and cmake
     export CHPL_AUX_FILESYS=none
 
+    # We default to CHPL_LIBFABRIC=system for EX.  We need to point to
+    # a libfabric install for the builds.  On EX a module will supply
+    # this but on XC we have to reference our own.
+    if ! pkg-config --exists libfabric ; then
+      export PKG_CONFIG_PATH=${PKG_CONFIG_PATH:+${PKG_CONFIG_PATH}:}/cray/css/users/chapelu/libfabric/install/cray-xc/lib/pkgconfig
```

Comment:
Can/should we just fallback to the system libfabric instead of a cray/css build?

Reply:
Yes, we'll do that when we build the module natively on EX, but there is no system libfabric on an XC cross-build system.

Reply:
Sorry, typo (that made it the exact opposite of what i wanted to say) -- should be fall back to the bundled libfabric if there's no system one, instead of using /cray/css?

Reply:
No, because then the 'libfabric' component of the runtime unique config path will say to use the bundled one instead of the system one.  One could imagine pre-building the bundled one and setting `PKG_CONFIG_PATH` to point to it since after all the only thing we need it for during the module build is its `.h` files, but that would be a lot of special-case code in the module build infrastructure, because the chapel repo doesn't even get cloned until within the scripts themselves.  And I view cross-building the EX module on XC as a bit of a stopgap, which we should put as little effort into as possible.

Reply:
Ah, I see. Alright, unfortunate, but makes sense.